cmd/login.go

```diff
@@ -53,10 +59,17 @@ func login(cmd *cobra.Command, args []string) {
 			username = strings.TrimRight(scanner.Text(), "\r\n")
 		}
 	}
-	fmt.Print("password: ")
-	password, err := terminal.ReadPassword(int(os.Stdin.Fd()))
-	if err != nil {
-		log.Fatal(err)
+
+	// try to get password from keyring, it's okay if we can't find it (i.e: ignore error)
+	password, _ := keyring.Get(keyringUser, keyringService)
```

Comment:
this did not work for me: when retrieving and printing the error, I have the following message: `secret not found in keyring`. I double-checked the account and service thatI used when running the `security add-generic-password` command, so I'm a bit lost... (BTW, if you add a space before the command, it won't be recorded in the session history)

Reply:
yep sorry I need to send an update, i have miss placed the arguments between service and user,

Reply:
doh, of course! I did not think about checking the order of the args in the `keyring.Get()` func call 🤦‍♂️ It works much better now ;)

Reply:
That was my bad :), my cli test cmd was inverted :) this is what you get trying to send a patch during the holiday periods ;)